react/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,33 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(arg) {
+  if (arguments.length === 1) {
```

Comment:
You don't need to use `arguments`. That's "the old way". You can use the spread operator on the parameters of your function to get the arguments as an array. Like `function min(...arg)`